recipes/nsls2-analysis/meta.yaml

```
@@ -0,0 +1,212 @@
+{% set name = "nsls2-analysis" %}
+{% set year = "2021" %}
+{% set cycle = "2" %}
+{% set version = "0" %}
+
+# Special pins:
+{% set dask_pin = "2021.4.0" %}
+{% set mkl_pin = "2019.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ year }}C{{ cycle }}.{{ version }}
+
+build:
+  number: 0
+  skip: true  # [py<37 or not linux]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - amostra <=1.0
+    - ansiwrap
+    - area-detector-handlers
+    - arvpyf
+    - attrs >=18.0
+    - black
+    - bluesky >=1.6.7
+    - bluesky-kafka  # [not win]
+    - bluesky-live
+    - boto3
+    - chxtools
+    - conda-pack
+    - csxtools
+    # TODO: check that the latest 'distributed' package works with the latest
+    # 'dask' package. Currently we are pinning the both libraries to have the
+    # same version.
+    - dask  # ={{ dask_pin }}
+    - databroker >=1.2.0
+    - databroker-pack
+    - distributed  # ={{ dask_pin }}
+    - doi2bib
+    # - edrixs
+    - eiger-io
+    - event-model >=1.17.0
+    - fabio
+    - ffmpeg >=4.0
+    - flake8
+    - globus-sdk
+    - graphviz
+    - grid-strategy
+    - h5py >=2.9.0
+    - hdf5-external-filter-plugins-bitshuffle
+    - hdf5-external-filter-plugins-lz4
+    - hxntools >=0.5
+    - igor
+    - inflection
+    # - intel-openmp ={{ mkl_pin }}
+    - ipykernel
+    - ipympl >=0.1.1
+    - ipython >=7.20.0
+    - ipywidgets >=7.2.1
+    - isort
+    - ispyb
+    - isstools
+    - jedi
+    - jupyter
+    - jupyterlab
+    - legacy-suitcase
+    - line_profiler
+    - lixtools
+    - lmfit
+    - lxml
+    - matplotlib >=3.1.0,!=3.3
+    - memory_profiler
+    - mendeleev
+    # - mkl ={{ mkl_pin }}
+    - modestimage
+    - napari >=0.3.7
+    - natsort
+    - netcdf4
+    - nexpy >=0.12.7
+    - nodejs
+    - nsls2-detector-handlers >=0.0.2
+    - nslsii >=0.1.1
+    - numexpr
+    - numpy >=1.14.0
+    - oct2py
+    - opencv
+    - openpyxl
+    - ophyd >=1.6.0
+    - oscars
+    - papermill
+    - pdfstream
+    - peakutils
+    - periodictable
+    - photutils
+    - pillow
+    - py4xs
+    - pycentroids  # [not win]
+    - pyepics >=3.4.2
+    - pyfai
+    - pyfftw
+    - pymongo >=3.7
+    - pypdf2
+    - pyqt >=5.9.0
+    - pyqtgraph
+    - pystackreg
+    - python
+    - python-graphviz
+    - pyxrf >=1.0.5
+    - pyzbar  # [not win]
+    - qt >=5.9.0
+    - redis-py
+    - reportlab
+    - requests
+    - sasview
+    - scikit-beam >=0.0.23
+    - scikit-learn
+    - scipy
```

One more simulation package:
```suggestion
    - scipy
    - shadow3
```